README.md

```diff
@@ -400,6 +400,28 @@ if (!self.penumbra) {
 }
 ```
 
+### Querying Penumbra browser support
+
+You can check if Penumbra is supported by the current browser by comparing `penumbra.supported(): PenumbraSupportLevel` with `penumbra.supported.levels`.
+
+```ts
+if (penumbra.supported() > penumbra.supported.levels.possible) {
+  // penumbra is partially or fully supported
+}
+
+/** penumbra.supported.levels - Penumbra user agent support levels */
+enum PenumbraSupportLevel {
+  /** Old browser where Penumbra does not work at all */
+  none = -0,
```

Comment:
what is -0?

Reply:
Same as zero, but theoretically detectably different. Support levels `none` & `possible` should be treated mostly the same.